test/index.js

```diff
@@ -66,7 +70,12 @@ test('absolute', function(t) {
 	};
 	clearResults('test/result/copy.css', 'test/imported');
 	return compareFixtures(t, 'should change existing assets path', rebaserOptions, postcssOptions)
-		.then(function () {
+		.then(function (result) {
```

Comment:
i don't like this repeating part of code with
```
t.equal(result.warnings()[0].toString(), [
```

I suggest to move this check to standalone method